src/main/java/com/mongodb/kafka/connect/sink/cdc/CdcHandler.java

```diff
@@ -40,4 +44,8 @@ public MongoSinkTopicConfig getConfig() {
   }
 
   public abstract Optional<WriteModel<BsonDocument>> handle(SinkDocument doc);
+
+  public List<WriteModel<BsonDocument>> createWriteModels(final SinkDocument doc) {
```

Comment:
This change is required because in the case of `truncatedArrays`, two operations are needed one to truncate the array and one to set a value.

By default the original handle method is called.